Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -1885,6 +1916,8 @@ class AnimatedTracking extends Animated {
 type CompositeAnimation = {
   start: (callback?: ?EndCallback) => void,
   stop: () => void,
+  reset: () => void,
+  startNativeLoop: (iterations?: ?number) => void,
```

Comment:
I see your point, but just startNativeLoop right? The ```reset``` is could be useful publicly?